src/Ch1/BMIDemo.java

```diff
@@ -0,0 +1,28 @@
+//Program to Compute Body Mass Index
+
+/**
+ * @Author Naresh

```

Comment:
@Author must be written as "@author". Java documentation have standard tags, so we have to follow as it is.